test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -3169,6 +3177,10 @@ class test_ucp_sockaddr_protocols_err_sender
             send_recv(sender(), receiver(), send_recv_type(), false, cb_type(),
                       sender_idx);
 
+            if (!has_rndv_lanes(sender())) {
+                UCS_TEST_SKIP_R("no RNDV lanes");
+            }
```

Comment:
why need this check now - how did it pass before?

Reply:
See here: https://github.com/openucx/ucx/pull/10230/files#r1846321667